lambda/src/main/resources/db/migration/V89__Insert_displayproperties_data.sql

```diff
@@ -1,6 +1,6 @@
 -- Insert data into the DisplayProperties table
 
-INSERT INTO "DisplayProperties" ("PropertyName","Attribute","Value","AttributeType") 
+INSERT INTO "DisplayProperties" ("PropertyName","Attribute","Value","AttributeType")
```

Comment:
You'll have to revert this change I think. The tests are passing because it's running `flywayMigrate` on a clean database but if you try to run it on an existing database, it fails because the checksum of this file doesn't match.

Reply:
Oh OK, I reverted it, thanks.